### ISSUE TITLE
Allow Multiple Images for Header and Footer Logos

### ISSUE THREAD

**Issue by Saixel** [IQSS/dataverse#10693]
## Background
In the platform, users can customize the appearance of their collection pages by adding logos to the header and footer. Currently, only one image can be uploaded for the "Logo Image" and one for the "Footer Image" on the main CAFE Dataverse collection page. University branding policies require that the BU and Harvard logos not be combined into a single file, which is a significant limitation.

## Issue
The current restriction of uploading only one image for header and footer logos prevents compliance with university branding policies that require multiple logos. This limits customization capabilities and adherence to institutional guidelines.

## Feature Request
Enable the ability to upload multiple images for header and footer logos on the Dataverse collection page. This would allow the alignment of multiple logos (e.g., BU, Harvard, and CAFE) to comply with university branding policies.

## Justification
Allowing multiple images for logos will increase the flexibility and customization of collection pages in Dataverse, ensuring compliance with institutional branding policies. This will enhance the professional appearance and brand consistency of Dataverse pages.

## Implementation Considerations
- Assess how multiple images for logos can be facilitated in the user interface.
- Ensure the platform can handle and store multiple logo images.
- Consider alignment options (left, center, right) for each logo image.

## Additional Context
Dataverse currently allows logo alignment to the left, center, and right. One potential solution would be to have the ability to upload three separate image files for each of these alignments.

**Comment by sbarbosadataverse** [comment]
Providing some feedback:

Based on my experiences with customizing logos (headers, footers) representing multiple institutions, it's easier to simple create an image of the combined logos (like the examples provided below), rather than try to make this a feature in the software. The theme/widget feature has numerous restrictions on sizing and placement and I've only known UX specialist to design this feature. Do we have a specialist on our team at the moment? @dliburd can you chime in on this request?

examples of customized logos: this is a complex example which would be really hard to do in the software. 

https://dataverse.harvard.edu/dataverse/CGIAR

**Comment by pdurbin** [comment]
@sbarbosadataverse thanks, the example you gave is great:

![Cgiar-Banner2](https://github.com/user-attachments/assets/6196fdb3-823c-4987-9bca-a766c4ed9675)

I agree it's probably better to ask the user to create complex images like this than to add a feature to Dataverse.

**Comment by dliburd** [comment]
I agree with Sonia. I also think there is a misunderstanding on university/corporate guidelines on logos. I reviewed Harvard University’s own brand guidelines. There is no rule against having the logo in the “same file.” Also, the Harvard logo can be used alongside other logos if there is enough space given to the HU logo. I think it’s fair to assume other organizations’ logo guidelines would be similar. If that wasn’t the case, it would be impossible to tweet flyer images with multiple sponsor logos and university logos.

Harvard’s standards are against modifying by changing the shape and colors of their logos. An all black or all white versions are acceptable in cases where the full colored logo doesn’t have enough contrast with the background. Most organization logo brand guidelines follow that logic.
